app/controllers/home_controller.rb

```diff
@@ -5,15 +5,11 @@ class HomeController < ApplicationController
   def index
     @projects =
       Rails.cache.fetch("home-index", expires_in: 24.hours) do
-        _projects =
-          [
-            Project.new(repo_name: "sinatra/sinatra", ci_type: :travis),
-            Project.new(repo_name: "jollygoodcode/twemoji", ci_type: :travis),
-            Project.new(repo_name: "winston/google_visualr", ci_type: :travis)
-          ]
-        _projects.map! { |project| ProjectDecorator.new(project) }
-        _projects.each(&:process_with)
-        _projects
+        [
+          Project.new(repo_name: "sinatra/sinatra", ci_type: :travis),
+          Project.new(repo_name: "jollygoodcode/twemoji", ci_type: :travis),
+          Project.new(repo_name: "winston/google_visualr", ci_type: :travis)
+        ].map! { |project| ProjectDecorator.new(project) }.each(&:process_with)
```

Comment:
Ok. I thought it returns something else.

:+1:

On Friday, 9 October 2015, Juanito Fatas <notifications@github.com> wrote:

> In app/controllers/home_controller.rb
> <https://github.com/jollygoodcode/dasherize/pull/24#discussion_r41595519>:
>
> > @@ -5,15 +5,11 @@ class HomeController < ApplicationController
> >    def index
> >      @projects =
> >        Rails.cache.fetch("home-index", expires_in: 24.hours) do
> > -        _projects =
> > -          [
> > -            Project.new(repo_name: "sinatra/sinatra", ci_type: :travis),
> > -            Project.new(repo_name: "jollygoodcode/twemoji", ci_type: :travis),
> > -            Project.new(repo_name: "winston/google_visualr", ci_type: :travis)
> > -          ]
> > -        _projects.map! { |project| ProjectDecorator.new(project) }
> > -        _projects.each(&:process_with)
> > -        _projects
> > +        [
> > +          Project.new(repo_name: "sinatra/sinatra", ci_type: :travis),
> > +          Project.new(repo_name: "jollygoodcode/twemoji", ci_type: :travis),
> > +          Project.new(repo_name: "winston/google_visualr", ci_type: :travis)
> > +        ].map! { |project| ProjectDecorator.new(project) }.each(&:process_with)
>
> each will return the caller:
>
> [1,2,3].each(&:odd?)# => [1, 2, 3]
>
> So still works.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jollygoodcode/dasherize/pull/24/files#r41595519>.
>


-- 
Cheers,
Winston

*Founder / Principal Engineering Consultant*

*Jolly Good CodeT: @winstonywP: +65 9437 1845*